projects/vdk-plugins/airflow-provider-vdk/vdk_provider/hooks/vdk.py

```diff
@@ -1,75 +1,110 @@
+
+        self.__execution_api = ApiClientFactory(self.base_url).get_execution_api()
+
         # hook methods will use HttpHook's `run_with_advanced_retry` method and pass the _retry_dict as a parameter
-        self._retry_dict = {
-            "stop": stop_after_attempt(retry_limit),
-            "sleep": sleep(retry_delay),
-        }
-        self.headers = {
-            "content-type": "application/json",
-            "authorization": f"Bearer {Authentication().read_access_token()}",
-        }
-
-    def start_job_execution(self) -> None:
+        # self._retry_dict = {
+        #     "stop": stop_after_attempt(retry_limit),
+        #     "sleep": sleep(retry_delay),
+        # }
+        # self.headers = {
+        #     "content-type": "application/json",
+        #     "authorization": f"Bearer {Authentication().read_access_token()}",
+        # }
+
+    def start_job_execution(self, **request_kwargs) -> None:
         """
-        API path: POST /data-jobs/for-team/{team_name}/jobs/{job_name}/deployments/{deployment_id}/executions
+        Triggers a manual Datajob execution.
 
-        :return: None
+        :param: request_kwargs: Request arguments to be included with the HTTP request
 
         self.method = "POST"
-
         endpoint = f"/data-jobs/for-team/{self.team_name}/jobs/{self.job_name}/deployments/{self.deployment_id}/executions"
+
         self.run_with_advanced_retry(
-            self._retry_dict, endpoint=endpoint, headers=self.headers, json={}
+            self._retry_dict,
+            endpoint=endpoint,
+            headers=self.headers,
+            extra_options=self.extra_options,
+            json={},
         )
         """
-        self.method = "POST"
-
-        return None  # included this to ignore faulty codacy check
+        execution_request = DataJobExecutionRequest(
+            started_by=f"vdk-control-cli",
+            args=request_kwargs,
+        )
+        _, _, headers = self.__execution_api.data_job_execution_start_with_http_info(
+            team_name=self.team_name,
+            job_name=self.job_name,
+            deployment_id=self.deployment_id,
+            data_job_execution_request=execution_request,
+        )
+        log.debug(f"Received headers: {headers}")
 
     def cancel_job_execution(self, execution_id: str) -> None:
         """
-        API path: DELETE /data-jobs/for-team/{team_name}/jobs/{job_name}/executions/{execution_id}
+        Cancels a Datajob execution.
 
         :param execution_id: ID of the job execution
-        :return: None
 
         self.method = "DELETE"
-
         endpoint = f"/data-jobs/for-team/{self.team_name}/jobs/{self.job_name}/executions/{execution_id}"
+
         self.run_with_advanced_retry(
-            self._retry_dict, endpoint=endpoint, headers=self.headers, json={}
+            self._retry_dict,
+            endpoint=endpoint,
+            headers=self.headers,
+            extra_options=self.extra_options,
+            json={},
```

Comment:
This seems the previous version, Is it needed?

Reply:
dropped